openvm/src/customize_exe.rs

```diff
@@ -61,17 +61,19 @@ impl From<powdr_autoprecompiles::constraint_optimizer::Error> for Error {
     }
 }
 
-pub fn customize<P: IntoOpenVm>(
+pub fn customize(
     OriginalCompiledProgram {
         mut exe,
         sdk_vm_config,
-    }: OriginalCompiledProgram<P>,
+    }: OriginalCompiledProgram,
     labels: &BTreeSet<u32>,
```

Comment:
airs and bus_map are not passed anymore, they are generated inside this function based on the config